src/Schema/Values/ContextModule.ts

```diff
@@ -70,6 +70,7 @@ export enum ContextModule {
   curatedHighlightsRail = "curatedHighlightsRail",
   curatedTrendingArtistsRail = "curatedTrendingArtistsRail",
   curatorsPicksEmergingRail = "curatorsPicksEmergingRail",
+  currentAlertArtworkMatches = "currentAlertArtworkMatches",
```

Comment:
**suggestion**: Following up on [this Slack thread](https://artsy.slack.com/archives/C05EQL4R5N0/p1700049456288239), what do you think of `alertDetails`, `alertFilters`, and `alertConfirmation` as new context modules to add here?

Reply:
missed this one
adding now